### PR TITLE
VIH-10852 update to court rules page on participant journey

### PR DESCRIPTION
**Pull request by marcogagliardi**
### Jira link
[VIH-10852](https://tools.hmcts.net/jira/browse/VIH-10852)

### Change description
Added abuse-warning text

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/assets/i18n/en.json [hunk 1]
```diff
@@ -218,6 +218,7 @@
     "are-not-allowed-to-smoke-including-e-cigarettes": "are not allowed to smoke (including e-cigarettes)",
     "must-be-alone-unless-you-have-the-judges-permission": "must be alone unless you have the judge’s permission to have someone with you",
     "must-ask-the-judges-permission-to-move-away-from-screen": "must ask the judge's permission if you need to move away from your screen",
+    "abuse-warning": "abuse will NOT be tolerated, anyone using threatening or aggressive behaviour against our staff or judges will  be removed from the hearing. You may also be reported to and arrested by the Police; you may be sent to Prison; your case may continue and be decided in your absence.",
```
Comment (oliver-scott) [inline VideoWeb/VideoWeb/ClientApp/src/assets/i18n/en.json:221]: I know this probably came straight from the ticket, but there are some typos in this:

_our staff or judges will  be_ -> there's an extra space between _will_ and _be_
_Police_ -> should be lowercase p (police)
_Prison_ -> should be lowercase p (prison)

File: VideoWeb/VideoWeb/ClientApp/src/assets/i18n/cy.json [hunk 2]
```diff
@@ -485,6 +485,7 @@
     "are-not-allowed-to-smoke-including-e-cigarettes": "ni chaniateir ichi ysmygu (gan gynnwys e-sigarets)",
     "must-be-alone-unless-you-have-the-judges-permission": "rhaid ichi fod ar ein pen ein hun oni bai bod y barnwr wedi rhoi caniatâd ichi gael rhywun gyda chi",
     "must-ask-the-judges-permission-to-move-away-from-screen": "rhaid ichi ofyn am ganiatâd y barnwr os byddwch angen symud i ffwrdd o'ch sgrin ",
+    "abuse-warning": "abuse will NOT be tolerated, anyone using threatening or aggressive behaviour against our staff or judges will  be removed from the hearing. You may also be reported to and arrested by the Police; you may be sent to Prison; your case may continue and be decided in your absence.",
```
Comment (oliver-scott) [inline VideoWeb/VideoWeb/ClientApp/src/assets/i18n/cy.json:488]: could we get the welsh for this?

**Review by oliver-scott — COMMENTED** [review]
looks good, left a couple of comments

File: VideoWeb/VideoWeb/ClientApp/src/assets/i18n/en.json (hunk 1 above)
Comment (marcogagliardi) [reply VideoWeb/VideoWeb/ClientApp/src/assets/i18n/en.json:221]: good spot. Fixed it

File: VideoWeb/VideoWeb/ClientApp/src/assets/i18n/cy.json (hunk 2 above)
Comment (marcogagliardi) [reply VideoWeb/VideoWeb/ClientApp/src/assets/i18n/cy.json:488]: Michael is working on it. We will have a different ticket to update the Welsh

**Review by oliver-scott — APPROVED** [review]

**Review by komaldabhi1509 — APPROVED** [review]

**Pull request merged**